lib/DockRenderer.vala

```diff
@@ -442,6 +442,9 @@ namespace Plank
 					var draw_value = position_manager.get_draw_value_for_item (item);
 					draw_item (item_cr, item, draw_value, frame_time);
 					draw_item_shadow (shadow_cr, item, draw_value);
+					// draw indicators
+					if (draw_value.show_indicator && item.Indicator != IndicatorState.NONE)
+						draw_indicator_state (main_cr, draw_value.hover_region, item.Indicator, item.State);
```

Comment:
You should put this in brackets, as specified by elementary code style documentations.
Otherwise looks good. 👍🏽️

Reply:
Will do, I didn't originally because I wanted to preserve the code style used here.